pkg/network/node/egressip.go

```diff
@@ -269,6 +270,12 @@ func (eip *egressIPWatcher) claimEgressIP(egressIP, egressHex string) error {
 
 			for _, addr := range addrs {
 				if addr.IP.String() == eip.localIP {
+					_, eip.localEgressNet, err = net.ParseCIDR(addr.IPNet.String())
```

Comment:
Why to run this for every claim? 
We know localIP during egressIPWatcher intialization. We could populate localEgressNet in Start() before calling the HostSubnet/NetNamespace watch.

Reply:
We don't, we only run it on the first claim. (There's an "`if eip.localEgressLink == nil`" around the whole thing.) I don't remember why I did it that way rather than initializing it from Start() but maybe I should change that...